blackbox/README.md

```diff
@@ -0,0 +1,19 @@
+# Stratis blackbox test run script
+
+The stratis-blackbox-run.sh script builds a set of testing RPM packages of the stratisd and stratis-cli master branches, runs a series of "blackbox" tests against the version of Stratis installed by these packages, and then uninstalls the pacakges.
+
+The blackbox test requires three scratch devices, loaded from the `/etc/stratis/test_config.json` file, which contains a JSON array of paths to the scratch devices.
+
+Example `/etc/stratis/test_config.json` file:
+
+```
+{
+    "ok_to_destroy_dev_array_key": [
+                                   "/dev/vdb",
+                                   "/dev/vdc",
+                                   "/dev/vdd",
+    ]
+}
+```
+
+The RPMs are built with spec files that originate from the source RPMs, with one key modification: the version number and changelog dates are set in the future, for testing purposes.
```

Comment:
This must be the RHEL src RPMs, I'm betting the Fedora ones are very different. Best to be specific here.

Reply:
Yes, that's a good point, so I'll say "RHEL source RPMs", but I'll also add a note that the test script is intended to run on a RHEL test system.  (I tried a run on a Fedora system, and the "dnf install" step failed because there is no available "rust-toolset" package for Fedora.)